recipes/folly/all/conanfile.py

```diff
@@ -1,14 +1,13 @@
-from conan.tools.microsoft import is_msvc, msvc_runtime_flag
-from conan.tools.build import can_run
-from conan.tools.scm import Version
-from conan.tools import files
 from conan import ConanFile
-from conans import CMake, tools
 from conan.errors import ConanInvalidConfiguration
-import functools
+from conan.tools.microsoft import is_msvc, msvc_runtime_flag
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rmdir, replace_in_file
+from conan.tools.build import can_run, check_min_cppstd, supported_cppstd
```

Comment:
```suggestion
from conan.tools.build import can_run, check_min_cppstd
```
Removing supported_cppstd usage